packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -111,22 +131,22 @@ void onDetachedFromActivity() {
   }
 
   @Override
-  public void onMethodCall(MethodCall call, MethodChannel.Result result) {
+  public void onMethodCall(@NonNull MethodCall call, @NonNull MethodChannel.Result result) {
```

Comment:
It looks like some of the unit tests are failing with an IllegalArgumentException on calls to onMethodCall. 

I've had some difficulties getting the these tests to run locally, but given that the only change was to add @NonNull I'm assuming that some of the tests were passing in null somehow?

Reply:
Hmm the tests are definitely not passing in null. I've confirmed that the tests succeed on the commit before this one and not on this one, but I'm somewhat at a loss as to what causes this error:
```
Please provide Old SKU purchase information(token/id) or original external transaction id, not both.
java.lang.IllegalArgumentException: Please provide Old SKU purchase information(token/id) or original external transaction id, not both.
	at com.android.billingclient.api.BillingFlowParams$SubscriptionUpdateParams$Builder.build(com.android.billingclient:billing@@5.2.0:7)
	at com.android.billingclient.api.BillingFlowParams$Builder.build(com.android.billingclient:billing@@5.2.0:42)
	at io.flutter.plugins.inapppurchase.MethodCallHandlerImpl.launchBillingFlow(MethodCallHandlerImpl.java:331)
	at io.flutter.plugins.inapppurchase.MethodCallHandlerImpl.onMethodCall(MethodCallHandlerImpl.java:150)
	at io.flutter.plugins.inapppurchase.MethodCallHandlerTest.launchBillingFlow_null_AccountId_do_not_crash(MethodCallHandlerTest.java:292)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	...
```